examples/example_pump_data_model/data_models/1.Pump.container.yaml

```diff
@@ -0,0 +1,48 @@
+externalId: Pump
+name: Pump
+space: '{{space}}'
```

Comment:
We should already now follow the proper pattern for handling spaces:
1. Global spaces are used as core containers and views to be extended (e.g. cdf_core)
2. Customer extended governed/global containers, views, and data models should live in a read-only "custom_data_model" space with read-only
3. Data instances should be ingested into a separate data space referring to the customer extended views

Even more importantly from a migration point of view, we need an 1.5 where Asset and related modeling entities should live in a non-system space (as they are not sanctioned as Cognite system entities), but where we don't want variance across customers. I.e. we should establish a "global_templates" space that only contains template-models that are managed by Cognite on top of the global, system models, but separate from the customer's own data models.